app/views/exports/hamilton_gem_x_5p_chip_loading.csv.erb

```diff
@@ -35,11 +37,27 @@
   ancestral_plate_wells = @ancestor_plate.wells.index_by(&:location)
   rows_array = []
   each_source_metadata_for_plate(@plate) do |src_barcode, src_location, dest_well|
-    number_of_samples =  ancestral_plate_wells[src_location].aliquots.length
-    resuspension_volume = (number_of_samples * required_number_of_cells * wastage_factor).to_f / chip_loading_concentration
+    src_well = ancestral_plate_wells[src_location]
+    number_of_samples =  src_well.aliquots.length
+
+    # Use the pool wells number_of_cells_per_chip_well if available, otherwise error.
+    # Value should have been calculated and stored on the pool well polymetadata.
+    well_poly_metadatum = src_well.poly_metadatum_by_key(number_of_cells_per_chip_well_key)
+
+    # raise error if polymetadata is missing
+    raise "Missing poly metadata for number of cells per chip well for #{src_barcode} #{src_location}, " \
+            'cannot generate driver file' unless well_poly_metadatum
+
+    # extract value from the polymetadata
+    number_of_cells_per_chip_well = well_poly_metadatum&.value.to_f.nonzero?
+
+    # calculations
+    resuspension_volume = (number_of_samples * required_number_of_cells_per_sample_in_pool * wastage_factor).to_f / desired_chip_loading_concentration
     source_well_volume = resuspension_volume - volume_taken_for_cell_counting
-    sample_volume = desired_number_of_cells_per_chip_well.to_f/chip_loading_concentration 
+    sample_volume = number_of_cells_per_chip_well/desired_chip_loading_concentration
```

Comment:
Could add a comment saying `# amount to pick`, since I think the column is ambiguously named.

Reply:
done